assets/javascript/apps/pipeline/nodes/GetInputWidget.tsx

```diff
@@ -106,7 +106,7 @@ export const getNodeInputWidget = (param: InputWidgetParams) => {
   }
 
   const allowedInNode = nodeTypeToInputParamsMap[param.nodeType];
-  if (param.name == "name" || (allowedInNode && !allowedInNode.includes(param.name))) {
+  if (param.name == "name" || param.name == "history_mode" || (allowedInNode && !allowedInNode.includes(param.name))) {
```

Comment:
I don't think this change is necessary since `history_mode` doesn't appear in the `allowedInNode` mapping